### DISCUSSION TITLE
ZBM releases page on GitHub are misleading without ZFS version stated.

### DISCUSSION THREAD

**Discussion by isopix** [zbm-dev/zfsbootmenu#692] [Feature Requests]
We don't have any information about ZFS version used in pre-built images of [ZFSBootMenu v2.3.0]

In case of ZFS it can be most important factor to use it or not (to be able to boot out latest shiny pool.

I also do feel ZBM is dead now with no upgrades that would include latest ZFS stable release.
It will be more important with the upcoming 2.3.0 OpenZFS release, that will have many awesome new features.

**Comment by classabbyamp** [comment]
You weren't misled, but you seem to have drawn the wrong conclusion because our version number is close to that of the imminent OpenZFS release. ZFSBootMenu is an independent project and uses independent versioning. You might have noticed that our release 2.3.0 predates the OpenZFS 2.3.0 release candidates by nine months.

We usually try to publish a release announcement that includes the ZFS and Linux kernel versions included in our binary images, but those are not part of the project changelog or the GitHub release notices. Typically, an announcement on the ZFS subreddit notes this information. If we overlooked an announcement for version 2.3.0, we apologize.

With each OpenZFS release, we confirm that our current release will boot the latest pools. If there are compatibility issues, we tag new releases. This tradition will continue with the release of OpenZFS 2.3.0. However, we do not release images built against unreleased versions of OpenZFS. If you would like to explore the OpenZFS 2.3.0 release candidates, you are expected to create your own, compatible ZFSBootMenu image.

While the ZFSBootMenu devs all have other responsibilities in their lives, we have been working on the project [periodically as we have time](https://github.com/zbm-dev/zfsbootmenu/commits/master/).

**Comment by isopix** [comment]
But why kernel and ZFS version aren't part of project changelog and why isn't it included in GitHub release notices?

Because you are writing this info to Reddit,it shouldn't be more work to include this on GitHub too.

It would be very useful to have this info on GitHub directly, because it's the place where project lives and while data on reddit/site can be outdated, it's not the case in case of GitHub(until development will be switched to some other place). And all other sites redirects to GitHub when downloading, so it sounds strange for me, that GitHub (which is central place to the project) is threaded like 3rd party citizen in case of releases.

**Reply by ahesford** [reply]
The reason is mostly that we automatically pull the release announcements from the changelog, and this information doesn't belong in the changelog. The changelog documents structural changes *in this project* and is completely disconnected from whatever happens to be built into the release images. It's also completely irrelevant for users of the project who build their own images.

In any case, you can always check the ZFS and Linux versions by grabbing our build container and poking around with `--entrypoint=/bin/bash`.
